config/sst_check_deprecated.m4

```diff
@@ -1,21 +0,0 @@
-AC_DEFUN([SST_CHECK_DEPRECATED],
-[
-  sst_check_hdf5_happy="yes"
-
-  AC_ARG_ENABLE([deprecated],
-    [AS_HELP_STRING([--(dis|en)able-deprecated],
-      [Enable or disable deprecated code (Default: enabled)])],
-    [
```

Comment:
Did you forget to add a file? Or am I just misreading? I don't see where the `CHECK_PREVIEW_BUILD` function is defined.

Reply:
Yes.  Good catch.  File added.  I also pushed some other changes to get rid of the Component* version SubComponent constructor as a valid API.